app/data/15.3.yml.erb

```diff
@@ -3,6 +3,7 @@
   leap-switch: true
   arches:
   - name: x86_64
+    desc: Intel or AMD 64-bit servers, desktops, laptops and boards
```

Comment:
Maybe a small thing, but could we start with desktops and laptops and not servers?

Reply:
yup I agree, we should make it consistent even in Guillaume's case

Reply:
Actually let's keep Arm as it is, since the current description mirrors the actual userbase :-) I'll still take your suggestion on Intel.